packages/guides/resources/template/html/structure/layout.html.twig

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>{{ title }}{% if title!=null and guides_project.getTitle()!=null %} - {% endif %}{% if guides_project.getTitle() %}{{ guides_project.getTitle() }}{% endif %}</title>
+    <title>{{ title }}{% if title!=null and project.getTitle()!=null %} - {% endif %}{% if project.getTitle() %}{{ project.getTitle() }}{% endif %}</title>
```

Comment:
Can we now get this from the context? The `project` variable is conflicting with phpdocumentor core. That's why I renamed it. The context is available as `env` 

Extracting it from the context will help use to reduce the chance of conflicts.

Reply:
The project node is part of the data, not of the enviroment in my eyes?

Reply:
If you take it from the context, you can access it everywhere you might need it. Otherwise you have to add it in the renderers that need it, but get it from that same context?

Reply:
But please keep the name guides_project to prevent issues in phpDocumentor 😁